advanced/is-pattern-2/templates/analytics-dashboard/identity-server-analytics-dashboard-statefulset.yaml

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "fullname" . }}-analytics-dashboard-statefulset
+  namespace : {{ .Release.Namespace }}
+spec:
+  replicas: {{ default 2 .Values.wso2.deployment.wso2isAnalyticsDashboard.replicas }}
+  # strategy:
```

Comment:
@darsham3 let's leave the `strategy` section without keeping it commented out. No need to maintain commented out configurations.

Plus, we can assume that we are using the default StatefulSet update strategy, Kubernetes `RollingUpdate`s.